features/creating.feature

```diff
@@ -1,25 +0,0 @@
-Feature: Creating artefacts
-  In order to build GOV.UK
-  I want to create artefacts in panopticon
-
-  Background:
-    Given I am an admin
-    And I have stubbed publishing-api
-
-  Scenario: Creating artefacts directly in panopticon
-    When I visit the homepage
-    Then I should see a link to create an item
-
-    When I follow the link link to create an item
-    Then I should see the artefact form
-
-    When I fill in the form without a need
-     And I save, indicating that I want to go to the item
-
-    Then I should be redirected to Publisher
-     And an artefact should have be created with content_id
-
-  Scenario: Trying to create an artefact for a need that is already met
```

Comment:
I don't think we can do this anymore. Publisher has no concept of Needs, and Panopticon no longer has any say over the creation of content.